crates/sui-indexer-alt-framework/src/lib.rs

```diff
@@ -152,6 +158,8 @@ impl Indexer {
             .context("Failed to run pending migrations")?;
 
         let metrics = IndexerMetrics::new(registry);
+
+        // TODO (wlmyng): Defer additional, db-specific metrics as an input arg
```

Comment:
Documenting our conversation: The store should take care of its own metrics initialisation before it is passed into the `Indexer`, rather than having the metrics passed in alongside it. (Separately, the `Indexer` still needs to accept a `registry` to initialise its own metrics).